web/tests/markdown.test.js

```diff
@@ -733,7 +737,7 @@ test("message_flags", () => {
     message = {topic: "No links here", raw_content: input};
     markdown.apply_markdown(message);
     assert.equal(message.is_me_message, false);
-    assert.equal(message.flags.includes("wildcard_mentioned"), true);
+    assert.equal(message.flags.includes("topic_wildcard_mentioned"), true);
     assert.equal(message.flags.includes("mentioned"), false);
```

Comment:
Done

Reply:
Better way might be to implement a `assert_mentions(List[str])` function which asserts whatever is passed to it as is true, and whatever isn't is false.

Reply:
I think that would add an another layer of understanding - what `assert_mentions(List[str])` does i.e.
> it asserts whatever is passed to it as is true, and whatever isn't is false.

Currently it is easy to understand at one look.

But yeah, that is nothing super difficult to understand. Adding a comment will work.

Reply:
That's right. If the amount of code deduplicated seems high enough to justify implementing such a function, feel free to go ahead and implement if. Otherwise even the current situation is not too bad. I don't have a strong opinion here.